addons/web/static/src/views/form/form_controller.scss

```diff
@@ -305,6 +304,18 @@
 
     // Statusbar
     .o_form_statusbar {
+        position: relative;
+        top: unset;
```

Comment:
Why unset `top` ? :thinking:

Reply:
Just an oversight on my side, it works without it :ok_hand: